packages/redux-routine/src/runtime.ts

```diff
@@ -2,8 +2,11 @@
  * External dependencies
  */
 import { create } from 'rungen';
+// eslint-disable-next-line no-duplicate-imports
+import type { Control } from 'rungen';
```

Comment:
Can these imports be merged? Is there a benefit to keeping them separate?

Reply:
Because `Control` is used only as a type the compiler will complain about it not being an `import type`: https://github.com/WordPress/gutenberg/blob/7dc02162976848ff5cd4ed1924fa7aee809584d2/tsconfig.base.json#L24

Reply:
Oh never mind, it allowed it 🤷

Reply:
Right, this would be an error if there were _only_ a type import. But there's also a regular value import so it doesn't produce an error.